common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -308,7 +309,7 @@ public String getExchangeCurrencyCode() {
     public void setExchangeCurrencyCode(final String exchangeCurrencyCode) {
         prefs.edit().putString(PREFS_KEY_EXCHANGE_CURRENCY, exchangeCurrencyCode).apply();
     }
-
+    //todo: preference not used
     public boolean getExchangeCurrencyCodeDetected() {
         return prefs.getBoolean(PREFS_KEY_EXCHANGE_CURRENCY_DETECTED, false);
     }
```

Comment:
I don't know why Android Studio marks this as unused, but it is used in `WalletActivity.detectUserCountry()`

Is it because this method is in the `common` module, but it is used in the `wallet` module?